helper/packager.php

```diff
@@ -121,7 +121,7 @@ public function create_extension($data)
 		));
 
 		$component_data = $this->get_component_dialog_values();
-		$skeleton_files = array(
+		$skeleton_files[] = array(
```

Comment:
That doesn't work (you need unique keys), and the idea here is to not use array_merge inside of a loop, as it can be resource greedy.